cms/djangoapps/contentstore/rest_api/v1/serializers.py

```diff
@@ -12,7 +12,7 @@ class ProctoredExamSettingsSerializer(serializers.Serializer):
     enable_proctored_exams = serializers.BooleanField()
     allow_proctoring_opt_out = serializers.BooleanField()
     proctoring_provider = serializers.CharField()
-    proctoring_escalation_email = serializers.CharField()
+    proctoring_escalation_email = serializers.CharField(allow_blank=True)
```

Comment:
The [default](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/course_module.py#L902) for the field is `None`  and not the empty string. I don't think this should cause a problem, but I wanted to double check with you.

Reply:
yup, the current behavior stores this as null/None if not set